blueprints/app/index.js

```diff
@@ -48,9 +53,25 @@ module.exports = {
       blueprintOptions,
       embroider,
       lang: options.lang,
+      ciProvider: options.ciProvider,
     };
   },
 
+  files(options) {
+    if (this._files) {
+      return this._files;
+    }
+
+    let files = this._super();
+    if (options.ciProvider !== 'travis') {
+      this._files = files.filter((file) => file !== '.travis.yml');
+    } else {
+      this._files = files.filter((file) => file.indexOf('.github') < 0);
```

Comment:
I'm guessing we want to remove the whole `.github` folder and not just the ci.yml file